config.json

```diff
@@ -0,0 +1,541 @@
+{
+  "settings": {
+    "CloudDropAccessToken": {
```

Comment:
Relevant only to the publish-packages command.. But without specifying in the global settings, run tool throws error that the setting is not found..